src/xcode/ENA/ENAUITests/ENAUITests.swift

```diff
@@ -32,6 +32,8 @@ class ENAUITests: XCTestCase {
 		app.buttons["AppStrings.Onboarding.onboardingContinue"].tap()
 		XCTAssertTrue(app.buttons["AppStrings.Onboarding.onboardingContinue"].waitForExistence(timeout: 5.0))
 		app.buttons["AppStrings.Onboarding.onboardingContinue"].tap()
+		XCTAssertTrue(app.buttons["General.primaryFooterButton"].waitForExistence(timeout: 5.0))
```

Comment:
Constants? Helps to align with the flaky CI server load.
```suggestion
		XCTAssertTrue(app.buttons["AppStrings.Onboarding.onboardingContinue"].waitForExistence(timeout: .medium))
		app.buttons["AppStrings.Onboarding.onboardingContinue"].tap()
		XCTAssertTrue(app.buttons["General.primaryFooterButton"].waitForExistence(timeout: .medium))
```

Reply:
This time I saw it before merging, even went so far and updated every 5.0 to .medium
Thanks for reminding me!